op-service/txmgr/txmgr.go

```diff
@@ -118,7 +120,22 @@ func (m *SimpleTxManager) IncreaseGasPrice(ctx context.Context, tx *types.Transa
 		gasFeeCap = CalcGasFeeCap(head.BaseFee, gasTipCap)
 	}
 
```

Comment:
Thinking of possible options:
- we are below the basefee and need to bump to get above the basefee
- we are above the basefee and there is competition with the tip so we need to bump that
- we are below the basefee and going above the basefee means its not a 15% increase for RBF

I feel like we shouldn't error here and instead just ensure that we bump enough

Reply:
Switched to requiring the tip + feecap to both be bumped by a minimum of 15%.